grunttasks/l10n-htmllint.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.registerTask('l10n-htmllint', 'Lint l10n files', function () {
+    // grunt addition for htmllint
+  });
```

Comment:
Awesome, thanks @zaach!

Turns out I was looking at this wrong because something about that flow doesn't copy stuff into /dist/. But we should just be setting the task src to copy from `<%= yeoman.app %>/i18n/*/*.html` anyways.

@TDA: Looks like this should get the job done:

```js
  grunt.registerTask('l10n-htmllint', 'Lint l10n files', [
    'clean',
    'selectconfig:dist',
    'l10n-generate-pages',
    'l10n-json-to-html',
    'htmllint:l10n'
  ]);
```

### Output:
```
✗ grunt l10n-htmllint

Running "clean:tos_pp" (clean) task
>> 2 paths cleaned.

Running "clean:dist" (clean) task
>> 1 path cleaned.

Running "clean:app" (clean) task
>> 0 paths cleaned.

Running "l10n-create-json" task

Running "copy:strings" (copy) task
Copied 212 files

Running "po2json:all" (po2json) task

Running "po2json:template" (po2json) task

Running "l10n-generate-tos-pp" task

Running "clean:tos_pp" (clean) task
>> 0 paths cleaned.

Running "replace:tos_pp" (replace) task

Running "marked:tos_pp" (marked) task
>> 51 files created.

Running "copy:tos_pp" (copy) task
Copied 2 files

Running "l10n-compile-templates" task
Warning: Cannot read property 'fonts' of undefined Use --force to continue.

Aborted due to warnings.


Execution Time (2015-07-10 19:30:14 UTC)
loading tasks    2.3s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 62%
copy:strings    285ms  ▇▇▇▇▇▇▇▇▇ 8%
po2json:all     883ms  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 23%
replace:tos_pp   59ms  ▇▇ 2%
marked:tos_pp   145ms  ▇▇▇▇▇ 4%
Total 3.8s
```